WindowsClient/MainWindow.xaml.cs

```diff
@@ -53,11 +56,24 @@ public MainWindow()
         private void SetBackgroundWorker(DoWorkEventHandler DoWork, RunWorkerCompletedEventHandler RunWorkerCompleted)
         {
             BackgroundWorker worker = new BackgroundWorker();
-
             worker.DoWork += DoWork;
             worker.RunWorkerCompleted += RunWorkerCompleted;
-
             worker.RunWorkerAsync();
         }
+
+        // [ToDo.MoveToOwnClass] - temporarily this directory is here.
+        private void DirectoryChecker()
+        {
+            bool directory = Directory.Exists(@"C:\program files\PIM");
+            bool model3d = File.Exists(@"C:\program files\PIM\model3d_newdens.for");
+            bool kepler2 = File.Exists(@"C:\program files\PIM\kepler2.for");
+
+            if (directory != true)
+                Directory.CreateDirectory(@"C:\program files\PIM");
+            if (model3d != true)
```

Comment:
Use relative paths to work on all workspaces

Reply:
Instead of "C:\program files"...

Reply:
in a future commit, this path will be populated in a pre-installation configuration.